lib/minitest/assertions.rb

```diff
@@ -449,12 +449,13 @@ def assert_raises *exp
 
     ##
     # Fails unless +obj+ responds to +meth+.
+    # include_all defaults to false to match Object#respond_to?
 
-    def assert_respond_to obj, meth, msg = nil
+    def assert_respond_to obj, meth, include_all: false, msg = nil
```

Comment:
this isn't syntactically correct

AND it fails the tests once fixed

Reply:
Apologies, I have it fixed (https://github.com/minitest/minitest/commit/bdebcbb2965b8d8babfad4cc1fe2735d5c828eae) but I see you already merged support in https://github.com/minitest/minitest/commit/f908dda892e70b0389c61c12137a18be8e3fb179. Much appreciated.
![Screenshot from 2023-11-11 09-25-03](https://github.com/minitest/minitest/assets/1557529/f3a137a0-35e4-4065-aa62-9d0a2e4fd149)